docs/stage-1/README.md

```diff
@@ -9,8 +28,10 @@ The contracts used in this stage are:
 - [TokenManager.sol](../../contracts/TokenManager.sol): dictates which ERC20 tokens are accepted to exchange for sEURO, and stores exchange rate details
 - [SEuroCalculator.sol](../../contracts/SEuroCalculator.sol): calculates how many sEURO to exchange for the given collateral
 - [BondingCurve.sol](../../contracts/BondingCurve.sol): calculates the current price of sEURO in euros
+- [SEuro.sol](../../contracts/SEuro.sol): the ERC20-token representing the stable euro
 
 ### Relationship
 The relationship between the contracts is:
 1. **SEuroOffering** gets the exchange rate contracts from **TokenManager** and uses **SEuroCalculator** to calculate the exchange rate
-2. **SEuroCalculator** calculates the base exchange rate of ETH/ERC20 to euros, then gets the amount of sEURO that can be obtained with given euros
\ No newline at end of file
+2. **SEuroCalculator** calculates the base exchange rate of ETH/ERC20 to euros, then gets the amount of sEURO that can be obtained with given euros
+3. **BondingCurve** contains the sEURO token in **SEuro** and exposes the main APIs
```

Comment:
actually the bonding curve no longer uses the SEuro contract, so that dependency can be removed. good spot 😉

Reply:
good spot yourself! 😄